addons/l10n_latam_invoice_document/views/report_invoice.xml

```diff
@@ -1,38 +1,20 @@
 <?xml version="1.0" encoding="utf-8"?>
 <odoo>
-    <template id="report_invoice_document" inherit_id="account.report_invoice_document">
-
-        <xpath expr="//h2" position="attributes">
-            <attribute name="t-if">not o.l10n_latam_document_type_id.report_name</attribute>
```

Comment:
@jco-odoo  I am not sure but did not see any inherit from Chile to this report_invoice_document in 13.0. Also reviewed the PR for l10n_cl_edi and did not found a reference either.

@ren-odoo please could you confirm with Chile if they are using/extending l10n_latam_invoice_document.report_invoice_document in 13.0?

Reply:
@Danisan @suv-bmya 
Can you help us to confirm please if you are using l10n_latam_invoice_document.report_invoice_document in 13.0 please? 
Thank you

Reply:
In progress with this...

Reply:
I think this can be managed without a problem. I made tests today and it worked nevertheless, in odoo/odoo, report_invoice_document will serve exclusively as templating to be inherited in l10n_cl_edi or to have something like a proforma. This is because the concept of legal invoice will be only for l10n_cl_edi (as a consequence of being a graphic representation of the XML). 
The two tech problem I am solving is:
1) I need an extra section for the document references (as if they were lines), but I think I will add this section after this "h2".
2) the other is that paging is not permitted in Chile: I will need to fix a maximum of lines somehow.

Reply:
@Danisan will the pro forma look good?  (and the variables filled in in community will be shown there as well?